modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableViewTest.java

```diff
@@ -385,32 +389,94 @@ private String debug() {
         verifySelectionAfterPermutation();
     }
 
-    @Test public void testSelectionUpdatesCorrectlyAfterRootSetAll() {
+    @Test public void testSelectionUpdatesCorrectlyAfterRootReverseAndSetAll() {
         setupForPermutationTest();
-        reverseChildrenOrder(treeTableView.getRoot());
+        TreeItem<String> parentTreeItem = treeTableView.getRoot();
+        List<TreeItem<String>> childrenReversed = getReverseChildrenOrder(parentTreeItem);
+        parentTreeItem.getChildren().setAll(childrenReversed);
         verifySelectionAfterPermutation();
     }
 
-    @Test public void testSelectionUpdatesCorrectlyAfterChildSetAll() {
+    @Ignore("JDK-8248217")
```

Comment:
Question: did you mean to point to the umbrella Task or is there a more specific bug that you could point to (I think referring to the Task is fine if you aren't sure). This applies to all of the newly added `@Ignore`d tests.

Reply:
These ignored tests are related to removing and adding a TreeItem.
I have corrected the id to use JDK-8193442 for the tests that remove an item and created a new JBS issue JDK-8248389, and used it for the test of adding an item .